info-articles/Mapping in solidity - issue #19.md/code/readme.md

```diff
@@ -0,0 +1,19 @@
+## Read Me
+
+These contains a brief directions to run the code in this file.
+
+## Getting the Dependencies
+
+### Remix IDE
+
+You will be coding in Remix. While it has a desktop version, we recommend using the online version for maximal development experience.
+
+You can access it here [Remix](https://remix.ethereum.org/#optimize=false&runs=200&evmVersion=null&version=soljson-v0.8.17+commit.8df45f5f.js) 
+
+### Install Hardhat
+
+You will need Hardhat for the compiling and deployment. Run this on your terminal
+```
+nvm install latest

```

Comment:
This is a command to install the latest version of node, this doesn't install hardhat.

Reply:
Changed.